rust/template/main.rs

```diff
@@ -106,7 +103,7 @@ fn handle_cmd(handler: &Box<dyn IMTUpdateHandler<Value>>,
                 },
                 Some(rid) => rid as RelId
             };
-            let _ = db.lock().unwrap().format_rel(relid, &mut stdout());
+            let _ = hddlog.db.as_ref().map(|db|db.lock().unwrap().format_rel(relid, &mut stdout()));
```

Comment:
use the dump_table API?

Reply:
Actually, now that you mention this, I realize that the Rust `dump_table` and `commit_dump_changes` APIs should have been implemented to return iterators, which is a far more Rust-friendly approach that also makes it easier for the caller to observe the start and end of a transaction compared to getting callbacks. I need to move on to other things now, but I will create an issue so that one of us can eventually make this improvement.